lib/convection/model/template/resource/aws_s3_bucket_policy.rb

```diff
@@ -17,6 +17,7 @@ class S3BucketPolicy < Resource
 
           def_delegators :@document, :allow, :id, :version, :statement
           def_delegator :@document, :name, :policy_name
+          def_delegators :@document, :deny, :id, :version, :statement
```

Comment:
You should only need to add to the list in line 18. e.g. ` def_delegators :@document, :allow, :deny, :id, :version, :statement`